tests/containers/build-base

```diff
@@ -0,0 +1,29 @@
+FROM quay.io/centos/centos:stream9
+
+RUN dnf install -y dnf-plugin-config-manager
+# CRB is required for meson
+RUN dnf config-manager -y --set-enabled crb
+# EPEL is required for python3-flake8 and codespell
+RUN dnf install -y epel-release
+
+RUN dnf install \
```

Comment:
Could you please sort packages by their names to ease maintenance?

Reply:
Done.